### ISSUE TITLE
Shadow priest talent "Pain and Suffering" not working properly <- Really crucial to Shadow DPS

### ISSUE THREAD

**Issue by TEIWASTAKEN** [Duskhaven/Bug-Reports#646]
In the talent it states that Everytime your mind flay does damage it instantly refreshes the dot tick of your Shadow word pain.

But it fails to apply the damage of the shadow word pain.

What i mean by this is that it does refresh the dot, but it should do 1st tick damage on the refresh, this is how it works on Wrath of the Lich King and Wrath of the Lich King classic.

This is to combat that more haste you get more dot refreshes you get so more haste you get in this case leads in to less dot damage of shadow word pain and with enough haste the shadow word pain will never tick damage as the re apply of the mind flay is too quick for shadow word pain to get any ticks in.

TL;DR 

When ever Mind Flay refreshes the Dot of Shadow Word: Pain = it should do the initial dot damage of Shadow Word: Pain

More haste you get = Bigger DPS loss is to have those talents.

To fix this is just to make Shadow Word: Pain to do do instant tick damage on re application or initial application.

Pictures to proof:

Mind Flay Ticks:
https://i.imgur.com/Leggiph.png

Shadow Word: Pain Ticks
https://i.imgur.com/jiHTXVz.png

**Comment by Duskhaven** [comment]
Shadow Word: Pain ticks are scaling with Haste after the restart. This should hopefully fix the issue.
